daffodil-runtime1/src/main/scala/org/apache/daffodil/dpath/NodeInfo.scala

```diff
@@ -210,6 +210,25 @@ object NodeInfo extends Enum {
     }
   }
 
+  def fromClassTypeName(a: String) = {
+    a match {
+      // TODO figure out better solution
```

Comment:
Probably not a TODO. This solution is ok. The alternative we chatted about the other day with Steve also, .... can't recall what it was, but a comment that mentions that technique, but suggests we're doing it this way for simplicity sake is fine.

Reply:
Indeed. I'd intended to implement the classOf way post our discussion but it fell through the cracks. Also fun fact, while implementing I discovered errors if the static strings, so glad we're doing it this way instead.